tests/test_register.py

```diff
@@ -12,8 +12,7 @@
 
 @pytest.fixture()
 def register_settings(make_settings):
-    """Returns a factory function for settings.Settings for register"""
-
+    """Create settings.Settings for testing package registration."""
```

Comment:
I would revert this change. I think the the existing docstring more accurately describes the behavior of this fixture.